gitlab/assets/dashboards/overview.json

```diff
@@ -0,0 +1,655 @@
+            "tick": true,
+            "tick_edge": "bottom",
+            "tick_pos": "50%",
+            "title": false,
+            "type": "note",
+            "width": 31,
+            "x": 1,
+            "y": 39
+        },
+        {
+            "add_timeframe": true,
+            "auto_refresh": false,
+            "background_color": "gray",
+            "bgcolor": "gray",
+            "content": "Users",
+            "font_size": "18",
+            "height": 7,
+            "html": "Users",
+            "isShared": false,
+            "refresh_every": 30000,
+            "scaleFactor": 1,
+            "show_tick": true,
+            "text_align": "center",
+            "tick": true,
+            "tick_edge": "bottom",
+            "tick_pos": "50%",
+            "title": false,
+            "type": "note",
+            "width": 33,
+            "x": 79,
+            "y": 37
+        },
+        {
+            "add_timeframe": true,
+            "error": null,
+            "height": 12,
+            "isShared": false,
+            "legend": false,
+            "markers": [],
+            "sharedGlobalTime": {
+                "live_span": "1h"
+            },
+            "tile_def": {
+                "markers": [],
+                "requests": [
+                    {
+                        "q": "avg:gitlab.user_session_logins_total{*}",
+                        "style": {
+                            "palette": "dog_classic",
+                            "type": "solid",
+                            "width": "normal"
+                        },
+                        "type": "line"
+                    }
+                ],
+                "viz": "timeseries",
+                "yaxis": {
+                    "includeZero": true,
+                    "label": "",
+                    "max": "auto",
+                    "min": "auto",
+                    "scale": "linear"
+                }
+            },
+            "time": {},
+            "title": true,
+            "title_align": "left",
+            "title_size": "16",
+            "title_text": "User Logins ",
+            "type": "timeseries",
+            "viz": "timeseries",
+            "width": 33,
+            "x": 79,
+            "y": 46,
+            "yaxis": {
+                "includeZero": true,
+                "label": "",
+                "max": "auto",
+                "min": "auto",
+                "scale": "linear"
+            }
+        },
+        {
+            "add_timeframe": true,
+            "board_id": "52g-qkm-q4e",
+            "error": null,
+            "generated_title": "Avg of gitlab.method_call_duration_seconds.count over *",
+            "height": 13,
+            "isShared": false,
+            "legend": false,
+            "tile_def": {
+                "markers": [],
+                "requests": [
+                    {
+                        "q": "avg:gitlab.method_call_duration_seconds.count{*}",
```

Comment:
You're plotting the total number of method calls it's probably not a useful thing to have.
Also this metric will need a better visualisation, splitting by tag or something as I don't think it provides any value as is.